src/util/Color.cxx

```diff
@@ -58,7 +58,7 @@ float RGBAColor::saturation() const noexcept
 
 RGBAColor RGBAColor::fromHSLA(float h, float s, float l, uint8_t a)
 {
-  int k = static_cast<int>(h / 21) % 12;
+  int k = static_cast<int>(12 * h) % 12;
```

Comment:
This is a fix to fromHSLA that assumed hue was between 0-360. However, it was between 0-1